README.md

```diff
@@ -29,13 +30,12 @@ To get a local copy up and running follow these simple example steps.
 ## Author
 
 GitHub:
-[@sntgopl](https://github.com/sntgopl)
 [@Mov305](https://github.com/Mov305)
-Linkedin: Santiago Pons Lezica & Abdelrhman samy
+Linkedin: Abdelrhman samy
 
 ## Contributing
 
-Contributions, issues, and reviews are very welcome!
+Contributions, issues, and reviews are very welcome! 
 
 ## Support
 
```

Comment:
- [ ] Add all the sections like **Acknowledgements**, **License** as shown in the [sample](https://github.com/HAMMAS-SALEEM/ToDoList/blob/main/README.md) :blush:
- [ ] Please add your MIT.md file from this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :+1: This is your license file for your project so please take it seriously :+1: